### PR TITLE
German jobs

### PR DESCRIPTION
**Pull request by CodeAndChoke**
### What does this changes

Ignore venv/ in git. Added german jobs

### CODE REVIEW

**Comment by CodeAndChoke** [comment]
@fcurella Thanks for the review. I added the requested unit tests.

Best regards

File: faker/providers/job/de_DE/__init__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        'Koreanistiker',
+        'Mechaniker',
+        'Pfleger',
+        'Polizist',
+        'Pilot',
+        'Arzt',
+        'Krankenschwester',
+        'Medizininformatiker',
+        'Schornsteinfeger',
+        'Winzer'
```
Comment (fcurella) [inline faker/providers/job/de_DE/__init__.py]: Add a trailing comma here.

File: faker/providers/job/de_DE/__init__.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as BaseProvider
+
+
+class Provider(BaseProvider):
+    jobs = [
+        'Altenpfleger',
```
Comment (fcurella) [inline faker/providers/job/de_DE/__init__.py]: Is there a source for this list?

Comment (fcurella) [reply faker/providers/job/de_DE/__init__.py]: Could you add the source as comment above the list?

File: faker/providers/job/de_DE/__init__.py (hunk 1 above)
Comment (CodeAndChoke) [reply faker/providers/job/de_DE/__init__.py]: Thanks, added :)

File: faker/providers/job/de_DE/__init__.py (hunk 2 above)
Comment (CodeAndChoke) [reply faker/providers/job/de_DE/__init__.py]: Done :)

**Comment by fcurella** [comment]
Tests are failing because of an unrelated bug.

I've opened a PR to fix that test at https://github.com/joke2k/faker/pull/1004. Once that's merged in, You can rebase against master (or merge master into your branch) and push again.

**Comment by CodeAndChoke** [comment]
@fcurella thank you very much :) I will subscribe that issue

**Comment by fcurella** [comment]
#1004 is merged in

**Comment by CodeAndChoke** [comment]
> #1004 is merged in

Can you take a look now? Thanks :)

**Comment by fcurella** [comment]
Looks good! Thank you!

**Pull request merged**